<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BIBLIAteka</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listaKsiazek">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>300</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listaUzytkownikow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>300</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listaWypozyczonych">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImieNazwisko">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditID">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSzukaj">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dodajOsobeButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj użytkownika</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wypozyczButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Wypożycz książkę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oddajButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Oddaj książkę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wyswietlButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyświetl wszystkie wypożyczenia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="znajdzButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Szukaj</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imie">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Imię i nazwisko</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_id">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Id</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wysz_ksiazke">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyszukaj książkę</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lista_ks">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista dostępnych książek</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_uzytkownikow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista użytkowników</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wypozyczenia">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wypożyczone książki</string>
    </property>
   </widget>
   <widget class="QPushButton" name="usunUzytkownikaButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuń użytkownika</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
